integration/link-test.ts

```diff
@@ -452,32 +459,42 @@ describe("route module link export", () => {
     expect(stylesheetResponses.length).toEqual(1);
   });
 
-  it("adds links to the document", async () => {
-    let restoreJavaScript = await app.disableJavaScript();
-    let responses = app.collectResponses((url) =>
-      url.pathname.endsWith(".css")
-    );
+  test.describe("no js", () => {
+    test.use({ javaScriptEnabled: false });
 
-    await app.goto("/links");
-    await app.page.waitForSelector('[data-test-id="/links"]');
-    expect(responses.length).toEqual(4);
+    test("adds links to the document", async ({ page }) => {
+      let app = new PlaywrightFixture(appFixture, page);
+      let responses = app.collectResponses((url) =>
+        url.pathname.endsWith(".css")
+      );
 
-    await restoreJavaScript();
+      await app.goto("/links");
+      await page.waitForSelector('[data-test-id="/links"]');
+      expect(responses.length).toEqual(4);
+    });
   });
 
-  it("preloads assets for other pages and serves from browser cache on navigation", async () => {
+  // playwright doesn't support checking where a resource came from
```

Comment:
For any of the tests we currently have that playwright won't let us keep, let's delete those.